src/app/parcels/ActionBar/Statuses.tsx

```diff
@@ -16,9 +16,8 @@ import { fetchParcelStatuses } from "@/app/parcels/parcelsTable/fetchParcelTable
 export type StatusType = ParcelStatus[][number];
 
 const nonMenuStatuses: StatusType[] = [
-    "Driver Overview Downloaded", //Generated when driver overview pdf downloaded
     "Map Generated", //Generated when maps generated
-    "Out for Delivery",
```

Comment:
What was this used for in the past? If we update the database should we reorder this to match?

Reply:
the purpose of this array is just to store the statuses which should not be visible in the status dropdown (the dropdown which allows us to directly change the statuses of parcels)
"Out for Delivery" was already in this array, all I did was remove "Driver Overview Downloaded" as this is no longer a status, and move "Out for Delivery" from the bottom of the array to where the "Driver Overview Downloaded" entry was. the order of this array doesn't actually matter.